api/v1alpha/subschema/common.schema.json

```diff
@@ -93,6 +93,11 @@
       "description": "The minimum number of successful attempts for a suite to be considered as 'passed'.",
       "type": "integer",
       "minimum": 1
+    },
+    "smartRetry": {
```

Comment:
Wondering out loud:

If we use a marketing-ish term like _smartRetry_, should this rather be an object instead of a boolean? If it's a boolean, we won't be able to expand it in the future without a major version change (I can't say whether we'll ever need to or not).